src/components/allReviews/ReviewFilters/OrderBy.jsx

```diff
@@ -0,0 +1,11 @@
+
+
+const OrderBy = ({ setOrderBy, orderBy }) => {
+    return (
+        <button onClick={() => {
+            orderBy === 'order_by=DESC' ? setOrderBy('order_by=ASC') : setOrderBy('order_by=DESC')
+        }}
+        >{orderBy.slice(9)}</button>)
+}
```

Comment:
I would say since this button is in a form and is not a submit, it needs an aria-label to tell the user what the button is for. Look into aria-label conventions to find a good label idea